### PR TITLE
[Tchap-v2] Extern users management

### PR DESCRIPTION
**Pull request by Yoshiin**
Extern users management.
Change "Start a chat" right menu option.

- #173 
- #191 
- #206 
- #225 

See #201 for more informations.

### CODE REVIEW

File: src/components/views/rooms/MemberInfo.js [hunk 1]
```diff
@@ -530,11 +529,20 @@ module.exports = withMatrixClient(React.createClass({
...
+
+        return [addrObject];
+    },
+
     onNewDMClick: function() {
-        this.setState({ updating: this.state.updating + 1 });
-        createRoom({dmUserId: this.props.member.userId}).finally(() => {
-            this.setState({ updating: this.state.updating - 1 });
-        }).done();
+        onStartChatFinished(true, this._buildAddrObject(this.props.member.userId));
```
Comment (giomfo) [inline src/components/views/rooms/MemberInfo.js]: same request: please remove `this.props.member.userId `

File: src/components/views/rooms/MemberInfo.js [hunk 2]
```diff
@@ -763,6 +778,7 @@ module.exports = withMatrixClient(React.createClass({
                     { insertPillButton }
                     { ignoreButton }
                     { inviteUserButton }
```
Comment (giomfo) [inline src/components/views/rooms/MemberInfo.js]: please remove `inviteUserButton` here too

File: src/components/views/rooms/MemberInfo.js (hunk 1 above)
Comment (Yoshiin) [reply src/components/views/rooms/MemberInfo.js]: Done

File: src/components/views/rooms/MemberInfo.js (hunk 2 above)
Comment (Yoshiin) [reply src/components/views/rooms/MemberInfo.js]: Right

**Pull request merged**